### PR TITLE
add new job notification count attributes to job

### PR DESCRIPTION
**Pull request by Amtron**

### CODE REVIEW

File: python/jobs.py [hunk 1]
```diff
@@ -56,6 +56,12 @@ def __init__(self):
         self.json_property(int, 'design_status')
         self.json_property(int, 'payment_status')
         self.json_property(int, 'shipping_status')
+        self.json_property(int, 'unread_notifications_count')
+        self.json_property(int, 'unread_job_info_notifications_count')
```
Comment (DanielCollins) [inline python/jobs.py]: This one does not appear to exist. One named `production`, does, though.

File: python/jobs.py [hunk 2]
```diff
@@ -56,6 +56,12 @@ def __init__(self):
...
+        self.json_property(int, 'unread_job_drafting_notifications_count')
+        self.json_property(int, 'unread_job_production_notifications_count')
+        self.json_property(int, 'unread_job_shipping_notifications_count')
+        self.json_property(int, 'unread_job_invoicing_notifications_count')
```
Comment (DanielCollins) [inline python/jobs.py]: you should add a comment indicating that these attributes are not embedded by default. and perhaps also seperate them to below `completed` and `priority`. see the examples in users.py and notifications.py

File: python/jobs.py (hunk 1 above)
Comment (Amtron) [reply python/jobs.py]: I've added it in another PR inside private repo.

File: python/jobs.py (hunk 2 above)
Comment (Amtron) [reply python/jobs.py]: thanks.

File: python/jobs.py (hunk 1 above)
Comment (DanielCollins) [reply python/jobs.py]: hm, i haven't seen that one

Comment (Amtron) [reply python/jobs.py]: It's open now.

**Pull request merged**